### PR TITLE
Safe postcss values parser

### PR DESCRIPTION
**Pull request by romainmenke**

### CODE REVIEW

File: test/basic.expect.css [hunk 1]
```diff
@@ -45,3 +45,7 @@
 	color: hwb(0 0% 0% / 100%);
 	color: hwb(0 0% 0% / 50%);
 }
+
+.test-unparseable-color-function {
+	color: rgb(data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==);
```
Comment (romainmenke) [inline test/basic.expect.css]:
```suggestion
	color: rgb(; );
```

smaller test

File: test/basic.css [hunk 2]
```diff
@@ -45,3 +45,7 @@
 	color: hwb(0 0% 0% / 100%);
 	color: hwb(0 0% 0% / 50%);
 }
+
+.test-unparseable-color-function {
+	color: rgb(data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==);
```
Comment (romainmenke) [inline test/basic.css]:
```suggestion
	color: rgb(; );
```

smaller test

File: test/basic.preserve-true.expect.css [hunk 3]
```diff
@@ -69,3 +69,7 @@
 	color: hwb(0 0% 0% / 100%);
 	color: hwb(0 0% 0% / 50%);
 }
+
+.test-unparseable-color-function {
+	color: rgb(data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==);
```
Comment (romainmenke) [inline test/basic.preserve-true.expect.css]:
```suggestion
	color: rgb(; );
```

smaller test

**Review by romainmenke — COMMENTED** [review]
Maybe data url's don't make sense here and a minimal invalid value is better.

**Review by Antonio-Laguna — APPROVED** [review]

**Pull request merged**